packages/api-core/src/resources/organizations.js

```diff
@@ -73,24 +85,22 @@ export default class AvOrganizations extends AvApi {
       totalCount: totalOrgCount,
     } = organizationsData;
 
-    if (typeof permissionId !== 'string' && !Array.isArray(permissionId)) {
-      throw new TypeError(
-        'permissionId must be either an array of ids or a string'
-      );
-    }
-    if (typeof resourceIds !== 'string' && !Array.isArray(resourceIds)) {
-      throw new TypeError(
-        'resourceIds must be either an array of ids or a string'
-      );
-    }
+    let permissionIdsToUse = permissionIds || permissionId;
+    permissionIdsToUse = this.sanitizeIds(permissionIdsToUse);
+    const resourceIdsToUse = this.sanitizeIds(resourceIds);
 
     // resourceIds is passed as readOnly, convert so that we can use Array methods on it
-    const resourceIdsArray =
-      typeof resourceIds === 'string' ? [resourceIds] : resourceIds;
+    const resourceIdsArray = Array.isArray(resourceIdsToUse)
+      ? resourceIdsToUse
+      : [`${resourceIdsToUse}`];
```

Comment:
Should be good to remove template here and on line 99 since `sanitizeIds()` has been called